iwyu_regex.h

```diff
@@ -23,6 +23,13 @@ bool ParseRegexDialect(const char* str, RegexDialect* dialect);
 bool RegexMatch(RegexDialect dialect, const std::string& str,
                 const std::string& pattern);
 
+// Returns the result of replacing the first match of the `pattern` regular
```

Comment:
We don't have a tradition of backticks in comments in IWYU source, and that character is a little messy on international keyboards, so I'd like to avoid it. Could you phrase this as something like:
```
Returns input string with the first match of pattern replaced, for the given regex dialect.
```
to keep the markup to a minimum? Thanks

Reply:
Sounds good. Short and simple!
Backticks are useful to allow code analysis tool to be able to find and index symbols in comments. Unlike other common delimiters ('|' for instance), backticks do not clash with the rest of the C++ syntax.

Reply:
Also, if I may venture in "programming wisdom" territory, consistency (or tradition as you put it) shouldn't be used as a blocker for improving the code style or quality. Any coding style or language improvement has to go through a phase where newer code breaks consistency with old one (unless you batch-apply style improvements to the whole code base at once, but that's not scalable, if at all possible). Or course, style improvements should be discussed and approved, so you are right in pointing inconsistencies where they occur.

With this in mind, note that backticked symbols in comments are understood by clangd, so if IWYU started using them, developers would see immediate improvements to code indexing in their IDE.